video_xblock/static/js/videojs_event_plugin.js

```diff
@@ -141,6 +141,9 @@
     }
 
     window.xblockEventPlugin = XBlockEventPlugin;
-    window.videojs.plugin('xblockEventPlugin', xblockEventPlugin);
+    // add plugin if player has already initialized
```

Comment:
This code works fine. But not for brightcove. I've initiated this plagin in brightcove directly template.
https://github.com/raccoongang/xblock-video/blob/dev/video_xblock/static/html/brightcove.html#L24